### PR TITLE
[24] [Ch setup testing environment] 

### PR DESCRIPTION
**Pull request by NTElissa**
#### What does this PR do?
  This PR adds a testing environment to ensure all functionalities are tested before deployment.

#### Description of Task to be completed?
Installed and configured the testing environment using [Mocha, Chai, Sinon, Jest and Supertest].
Added tests for all functionalities to ensure they are working correctly.

#### How should this be manually tested?
Checkout this branch.
Run npm install to install all dependencies.
Run npm test to run all tests.
Check the console output for test results.

#### Any background context you want to provide?

Testing is an important part of the development process, as it helps ensure that all functionalities are working as expected and reduces the likelihood of bugs being introduced during deployment. By adding a testing environment, we can automate the testing process and catch any issues before they become a problem.
#### What are the relevant pivotal tracker/Trello stories?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# ecommerce-app-predators-bn
\ No newline at end of file
+# setup-continuous-integration-24
```
Comment (chanceDira) [inline README.md]: Why did you remove the title of the app?

File: test/sample.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function sum(a, b) {
+  return a + b;
+}
+
+function subtract(a, b) {
+  return a - b;
+}
+
```
Comment (chanceDira) [inline test/sample.js]: can you use the export default

**Review by chanceDira — CHANGES_REQUESTED** [review]
Good progress, just some feedback also since you are using jest can you create a jest config file?

File: tests/sample.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export default{
+ sum(a, b) {
+  return a + b;
```
Comment (ALVINdimpos) [inline tests/sample.js:3]: nice job ellisa

**Review by Muhet — APPROVED** [review]

File: tests/sample.test.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import calculator from "./sample";
+
+test("adds 1 + 2 to equal 3", () => {
+  expect(calculator.sum(1, 2)).toBe(3);
+});
+
```
Comment (ALVINdimpos) [inline tests/sample.test.js:6]: nice job

**Review by Muhet — APPROVED** [review]
Great Job

**Review by faid-terence — APPROVED** [review]

**Review by ALVINdimpos — APPROVED** [review]

**Review by nili-alu — APPROVED** [review]

**Review by chanceDira — APPROVED** [review]
Nice job @NTElissa

**Review by bryan936M — APPROVED** [review]
Excellent work

**Pull request merged**